### PR TITLE
Refactor promises to async/await and require to import

### PR DESCRIPTION
**Pull request by jh0ker**

### CODE REVIEW

File: lib/fragmentSender.js [hunk 1]
```diff
@@ -90,62 +91,66 @@ module.exports = function(chat, fragments, payload, initialMessage = '', initial
...
+        } else if (message.type === MEDIA) {
+            try {
+                await chat.sendAttachment(message.content);
+            } catch (e) {
+                errors.push({ message: 'Sending media failed', error: e });
             }
-        };
-        return sendNext();
-    });
+            return sendNext();
```
Comment (mraerino) [inline lib/fragmentSender.js]: This can be simplified to call `sendNext` after the if clauses

File: lib/metrics.js [hunk 2]
```diff
@@ -55,9 +59,3 @@ async function pushPrometheus() {
         promMetrics[key].reset();
     });
 }
-
-module.exports = {
-    reportCloudWatch,
-    pushPrometheus,
-    promMetrics,
-};
```
Comment (mraerino) [inline lib/metrics.js]: This should not be removed, but rather be the default export to not change behaviour

File: lib/pushData.js [hunk 3]
```diff
@@ -42,23 +44,19 @@ function assemblePush(push) {
...
+        console.log(`Failed to update push ${id} to delivered`, e.message);
+        throw e;
+    }
 }
-
-module.exports = {
-    getLatestPush,
-    assemblePush,
-    markSent,
-};
```
Comment (mraerino) [inline lib/pushData.js]: This should not be removed but become the default export to not change behaviour.

File: entrypoint/metrics.js [hunk 4]
```diff
@@ -37,21 +37,25 @@ function collectSubscriberMetrics(timing) {
...
+        const metrics = [
+            collectSubscriberMetrics(),
+            collectSubscriberMetrics('morning'),
+            collectSubscriberMetrics('evening'),
+            collectSubscriberMetrics('both'),
+        ];
+
+        for (const metric of metrics) {
+            await metric;
+        }
```
[the remark this answers is not in the document]
Comment (jh0ker) [reply entrypoint/metrics.js]: https://stackoverflow.com/a/45286517

[the remark this answers is not in the document]
Comment (mraerino) [reply entrypoint/metrics.js]: The second answer states a different opinion and the fiddle clearly shows this
https://stackoverflow.com/a/46895089/7257595

[the remark this answers is not in the document]
Comment (mraerino) [reply entrypoint/metrics.js]: Alright. ✅

**Pull request merged**